lib/ansible/modules/cloud/amazon/ecs_service.py

```diff
@@ -487,6 +487,7 @@
 from ansible.module_utils.aws.core import AnsibleAWSModule
 from ansible.module_utils.ec2 import ec2_argument_spec
 from ansible.module_utils.ec2 import snake_dict_to_camel_dict, map_complex_type, get_ec2_security_group_ids_from_names
+import ansible.module_utils.ec2 as ec2
```

Comment:
Instead of importing the entire module_util, could you please just add `AWSRetry` to the existing list of functions being imported from `ec2` in each file?

Reply:
Done.